source/_patterns/04-pages/home.json

```diff
@@ -389,7 +293,7 @@
               }
             },
             "formats": {
-              "list": ["Executable", "PDF"]
+              "list": [ "Executable", "PDF" ]
```

Comment:
These extra changes, while not entirely unwelcome, make it difficult to determine the changes that need our attention in review.

Reply:
The extra spaces are removed